tests/phpunit/CRM/Contribute/Form/Task/PDFLetterCommonTest.php

```diff
@@ -505,9 +505,8 @@ public function testPostProcessGroupByContact(): void {
     $this->assertEquals($html[2], $activities['values'][1]['details']);
     // Checking it is not called multiple times.
     // once for each contact create + once for the activities.
-    // & once for the token processor, for now.
     // By calling the cached function we can get this down to 1
-    $this->assertEquals(4, $this->hookTokensCalled);
```

Comment:
Test change reflects one less call to the token processor - from the comments less is more. This will be the removal of the 'categories' call